utils/src/main/scala/za/co/absa/enceladus/utils/schema/SparkUtils.scala

```diff
@@ -66,6 +67,8 @@ object SparkUtils {
     }
   }
 
+  def containsColumn(df: DataFrame, colName: String): Boolean = df.schema.exists(field => field.name.equalsIgnoreCase(colName))
```

Comment:
This might better belong to `SchemaUtils` and the method might accept `Schema` as the input parameter.

Reply:
SchemaUtils already contains such a functionality - `SchemaUtils.fieldExists`. But this method has the advantage of being available as implicit directly on the dataframe.

So as a middle ground, I have changed this method to be calling the above-mentioned `SchemaUtils.fieldExists`
```scala
def containsColumn(colName: String): Boolean = SchemaUtils.fieldExists(colName, df.schema)
```
Would you consider this acceptable or you object primarily to the fact that the column existence is queried on the dataframe instead of being queried on the schema instead?

Reply:
Hmm, on the other hand, its true, that then the implicit is too much. I am changing it to using the `SchemaUtils` directly.
```scala
SchemaUtils.fieldExists(Constants.EnceladusRecordId, conformedDF.schema))```

Reply:
Done.

Reply:
Sorry, I just read these comments. Whatever your decision about this, I'm happy. 😄